lib/smart_answer/calculators/paternity_adoption_pay_calculator.rb

```diff
@@ -19,6 +19,10 @@ def initialize(match_date)
       @matched_week = @expected_week
     end
 
+    def paternity_deadline
+      (adoption_placement_date + 55).strftime("%d-%m-%Y")
```

Comment:
I think using the `days` method would make this code a little clearer, e.g. `55.days`. Same comment for the  line in the [paternity pay calculator](https://github.com/alphagov/smart-answers/pull/5012/files#diff-5b59a5d5f9197ed17e221beddf090567af1c912c9d0e132df77e1681132b1ddaR17). What do you think?

Reply:
Makes sense 👍 Just only looking at base ruby docs so hadn't come across that